main.d

```diff
@@ -32,13 +32,18 @@ import sambamba.utils.common.ldc_gc_workaround;
 
 import utils.strip_bcf_header;
 import utils.lz4;
-import utils.version_ : VERSION;
+import utils.version_ : VERSION, LDC_VERSION_STRING, LDC_DMD_LLVM_VERSION_STRING, LDC_LDC_VERSION_STRING;
 
 import std.stdio;
 
 void printUsage() {
     stderr.writeln("sambamba " ~ VERSION);
     stderr.writeln();
+    stderr.writeln("This version was built with:");
+    stderr.writeln(LDC_VERSION_STRING);
```

Comment:
Please surround with `version(LDC) {}` otherwise it's misleading for development builds.